Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -568,12 +568,14 @@ bool BluetoothReal::OpenDevice(libusb_device* device)
     return false;
   }
 
+#ifndef __FreeBSD__
```

Comment:
This should probably be accompanied with a brief comment explaining why this is needed.

Reply:
done